pkg/providers/tinkerbell/stack/stack.go

```diff
@@ -119,6 +101,11 @@ func (s *Installer) Install(ctx context.Context, bundle releasev1alpha1.Tinkerbe
 		})
 	}
 
+	osiePath, err := getURIDir(bundle.Hook.Initramfs.Amd.URI)
```

Comment:
why is this path named as Initramfs? what about vmlinuz? This path is for both initrd and the kernel

Reply:
correct, on tinkerbell side, yes. but i was wondering more about whether that's verified anywhere on eks-a cli side in the bundle and stuff